plugins/org.eclipse.xtext.xbase/src/org/eclipse/xtext/xbase/compiler/output/ErrorTreeAppendable.java

```diff
@@ -58,7 +59,7 @@ public TreeAppendable append(JvmType type) {
 			if(unresolvedLink != null) {
 				INode linkNode = unresolvedLink.getThird();
 				if(linkNode != null) {
-					append(linkNode.getText().trim());
+					append(NodeModelUtils.getTokenText(linkNode).trim());
```

Comment:
calling .trim() is not necessary as getTokenText() returns already-trimmed strings.